modules/library/src/it/scala/zio/elasticsearch/HttpExecutorSpec.scala

```diff
@@ -424,9 +426,16 @@ object HttpExecutorSpec extends IntegrationSpec {
                            .upsert[TestDocument](firstSearchIndex, secondDocumentId, secondDocument)
                            .refreshTrue
```

Comment:
You can remove this refresh and just leave it for last call before search.

Reply:
It is now.